### DISCUSSION TITLE
Lable Syslog <remote> events

### DISCUSSION THREAD

**Discussion by martinzeifang** [wazuh/wazuh#15984] [General]
Hi everyone, 
I have two Systems that are sending almost similar events. My decoder files are mixing them sometimes up. Is it posible to lable incomming syslog files through the <remote> configuration in the ossec.conf file?

Thanks everyone
Greetings

**Comment by carlosdams** [comment]
Hi @martinzeifang ,

How are you getting these events into Wazuh? 
Are you sending the syslog events directly to the Wazuh Manager or you are using a Wazuh Agent as an intermediary?
Do these two systems have different IP addresses?

One recommended way is to have rsyslog and a Wazuh Agent as an intermediary [forwarding the syslog events](https://documentation.wazuh.com/current/cloud-service/your-environment/send-syslog-data.html ) and this way you can make rsyslog send the events to different files and these files will correspond to the values of the field `location` on the alerts

If you are sending the syslog events directly to the Wazuh Manager then these events will have the values for the agent field as it is the Wazuh Server (i.e. agent 000 and name of the Wazuh Server) in contrast to having a Wazuh Agent as an intermediary.
The value for the `location` field will be the IP address of the syslog system if you are sending the events directly to the Wazuh Server.

One simple way to always identify the syslog system is by saving queries filtering by a known field of these systems, like the IP address or the name of the file from the `location` field.

Something a little bit more advanced that you can do is to edit the `/usr/share/filebeat/module/wazuh/alerts/ingest/pipeline.json` and have a [processor](https://www.elastic.co/guide/en/elasticsearch/reference/7.10/set-processor.html) that replaces a certain value from the syslog events for one that makes it easier to identify these systems sending syslog.
